client/src/pages/signin.css

```diff
@@ -0,0 +1,100 @@
+.main {
```

Comment:
we're not going to use `.css` in our projects. Take a look at the Material UI [styles](https://material-ui.com/styles/basics/) page. We mainly want to use `makeStyles` and feed them into a `className` prop on each component